crowbar_framework/app/assets/stylesheets/variables.css.scss

```diff
@@ -6,6 +6,10 @@ $gray: lighten(#000, 33.5%);
 $gray-light: lighten(#000, 60%);
 $gray-lighter: lighten(#000, 93.5%);
 
+// Navbar colors
+// -------------------------
+$navbar-font-color: #bebebe;
```

Comment:
I think the name of this is pretty wrong as it's note related only to navbars, it's related more general to dropdowns as well.